Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,27 @@ class Program
     {
         static void Main(string[] args)
         {
+
+            // 1) Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/3
+            // 2) Take homework 2 code, place it in the forked branch and refactor it using functions.There should be as little duplicate code as possible (there should be functions for:
+            // -Calculating BMI(weight comes in kg, height comes in meters),
+            // -Prompt for input and converting it to int(print message for request, read console input and return converted input to int), 
+            // -Prompt for input and converting it to string(print message for request, read console input and return input),
+            // -Prompt for input and converting it to float(print message for request, read console input and return converted input to float).
+            // 3) Put all the function you made into the right places in Checks.cs class. Run tests, make sure you pass all the tests
+            // 4) Put all program.cs logic to Lesson3.cs.Call it from main function. Program class should look like this:
+
+            //public class Program
+            //        {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4909275)